setup.py

```diff
@@ -27,7 +27,7 @@ def get_version():
 
 
 install_requires = [
-    "boto3>=1.9.201",
+    "boto3>=1.24",
```

Comment:
This shouldn't be necessary. An up-to-date version is necessary to use `boto3.client("emr-serverless")`, but Moto itself should be installable/usable with older versions of boto3 as well.

Reply:
Rolled back the version update